app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    get_command = command.split(" ")
```

Comment:
There is no need to prove a space as a delimiter, it is a value by default. Remove it.